hack/run-ci-e2e-test.sh

```diff
@@ -86,12 +86,25 @@ fi
 # https://steps.svc.ci.openshift.org/help/ci-operator#release
 OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
 
-# Setup and run the operator
-if ! run_WMCO $OSDK; then
-  # Try to get the WMCO logs if possible
-  get_WMCO_logs
-  cleanup_WMCO $OSDK
-  exit 1
+# OO_INDEX points to the WMCO bundle index image created through https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/ workflow in CI.
+# Setup and run the operator only if $OO_INDEX is not set in CI, and the operator deployment is not available.
+OO_INDEX=${OO_INDEX:-}
+
+# SKIP_OPERATOR_RUN_CLEANUP will skip operator deployment and cleanup if set to true
+SKIP_OPERATOR_RUN_CLEANUP=${SKIP_OPERATOR_RUN_CLEANUP:-"false"}
+
+if [ -z "$OO_INDEX" ] && ! oc rollout status deployment windows-machine-config-operator -n openshift-windows-machine-config-operator; then
+ SKIP_OPERATOR_RUN_CLEANUP="true"
+fi
+
+if ! $SKIP_OPERATOR_RUN_CLEANUP; then
+  # Setup and run the operator
+  if ! run_WMCO $OSDK; then
```

Comment:
nit: This could be put on the same line, as one `if`
I'm still ok with it, though.